### ISSUE TITLE
LAN permission ios

### ISSUE THREAD

**Issue by sssssdddddmmmmm** [breeze2/dsu-controller-guides#36]
the app still doesn't ask for local network permission. this is required to use it with dolphin. on the guide it says make sure it has local network permissions but there's no way to do that unless the app itself prompts it.

**Comment by breeze2** [comment]
Do you mean you can't connect to the Dolphin emulator?

**Comment by sssssdddddmmmmm** [comment]
> Do you mean you can't connect to the Dolphin emulator?

yes, since the app doesn't request permission to access local networks. is it possible for the app to be updated to request this permission on ios? this would most likely solve the problem as it is the only step in your troubleshooting that i have not tried since it is impossible.

**Comment by breeze2** [comment]
Have you try to connect with DSUManager？

**Comment by breeze2** [comment]
Please tell me what you put in the fields of DSU Server address.

**Comment by sssssdddddmmmmm** [comment]
unfortunately I can't use DSUManager since I only have a mac, and for the server address I put the exact IP and port found in the DSUController app.

**Comment by breeze2** [comment]
Please confirm that your mac and phone are in the same LAN. Do their IPs have the same prefix?
